src/devtools/client/debugger/src/components/A11yIntention.js

```diff
@@ -3,28 +3,22 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 //
-import React from "react";
+import React, { useState } from "react";
 
-export default class A11yIntention extends React.Component {
-  state = { keyboard: false };
+export default function A11yIntention({ children }) {
+  const [isKeyboard, setIsKeyboard] = useState(false);
 
-  handleKeyDown = () => {
-    this.setState({ keyboard: true });
-  };
+  handleKeyDown = () => setIsKeyboard(true);
```

Comment:
:hand: Both of these callbacks probably need a `const ` in front.

Reply:
Good catch!